code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py

```diff
@@ -0,0 +1,15 @@
+//binomial coefficient in python 3
```

Comment:
Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

Reply:
@yatharthshahjpr The comment needs to start with a #